cmd/ctr/commands/plugins/plugins.go

```diff
@@ -37,6 +37,14 @@ var Command = cli.Command{
 	Name:    "plugins",
 	Aliases: []string{"plugin"},
 	Usage:   "provides information about containerd plugins",
+	Subcommands: []cli.Command{
+		listCommand,
+	},
+}
+
+var listCommand = cli.Command{
+	Name:  "list",
```

Comment:
Alias with `ls` like other list commands?

Reply:
Thought of that just as I clicked `Approve` :-}  I think that's a good suggestion--can we get that small change and then we can merge?

Reply:
Sure, commit updated